zerver/lib/upload.py

```diff
@@ -165,6 +168,23 @@ def upload_image_to_s3(
 
     key.set_contents_from_string(force_str(contents), headers=headers)
 
+def get_total_qouta(user):
+    # type: (UserProfile) -> int
+    uploads = Attachment.objects.filter(owner=user)
+    total_quota = 0
+
+    for item in uploads:
+        total_quota += item.size
+    return total_quota
+
+def check_upload_quota(user, uploaded_file_size):
```

Comment:
Maybe call this function `within_upload_quota`?  We usually use `check_foo` for a function that will raise an exception on failure, and regardless the new name would suggest a bool.